srcpkgs/libtickit/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'libtickit'
 pkgname=libtickit
-version=0.4.1
+version=0.4.3
 revision=1
 build_style=gnu-makefile
 make_build_args="LIBTOOL=${XBPS_CROSS_BASE}/usr/bin/libtool"
```

Comment:
Removing `make_build_args` and `make_install_args`, and applying my other suggestion below, fixed cross-compilation locally for me.